eng/testing/performance/performance-setup.ps1

```diff
@@ -100,6 +101,10 @@ if ($iOSNativeAOT) {
     $Configurations += " iOSStripSymbols=$iOSStripSymbols"
 }
 
+if ($HybridGlobalization -eq "True") {
```

Comment:
Given the `$HybridGlobalization` is _switch_, I think `if ($HybridGlobalization) {...` is fine.

Reply:
Going to keep this as checking against True so that it is only included when True, not when False.